staticConfig/logsToElastic.conf

```diff
@@ -125,10 +125,8 @@ filter {
 "^/organizations/[^/]+/approve$" => "/organizations/{organizationId}/approve"
 "^/organizations/[^/]+/aliases$" => "/organizations/{organizationId}/aliases"
 "^/organizations/[^/]+$" => "/organizations/{organizationId}"
-"^/organizations/[^/]+/aliases$" => "/organizations/{alias}/aliases"
```

Comment:
hmm...there were two of them.  Should we make an issue to make this more restful?

Reply:
Added to https://github.com/dockstore/dockstore/issues/1845